include/lemlib/util.hpp

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+namespace lemlib {
+template <typename T> T sgn(T val) { return (T(0) < val) - (val < T(0)); }
```

Comment:
this should be constexpr
```suggestion
template <typename T> constexpr T sgn(T val) { return (T(0) < val) - (val < T(0)); }
```